include/proxy-wasm/wasm.h

```diff
@@ -222,6 +228,7 @@ class WasmBase : public std::enable_shared_from_this<WasmBase> {
   std::unique_ptr<ShutdownHandle> shutdown_handle_;
   std::unordered_map<std::string, std::string>
       envs_; // environment variables passed through wasi.environ_get
+  WASIFileSystem fs_;
```

Comment:
let's have it as unique ref
```suggestion
  std::unique_ptr<WASIFileSystem> fs_;
```